apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -601,7 +669,7 @@ def parse(element_html, data):
             indent = int(answer["indent"] or 0)
             answer_code += (
                 ("    " * indent)
-                + lxml.html.fromstring(answer["inner_html"]).text_content()
+                + lxml.html.fromstring(answer["inner_html"]).text_content()  # type: ignore
```

Comment:
What does pyright say without the ignore?

Reply:
```
/PrairieLearn/apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py:668:62 - error: Cannot access member "text_content" for type "_Element"
    Member "text_content" is unknown (reportGeneralTypeIssues)
```

Reply:
Weird.